site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -501,6 +501,13 @@ include::../shared/_advanced-edition-blurb-api.adoc[]
 +
 :advanced_feature!:
 
+[field]#sourceApplicationId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.43.0#::
+The optional Id of an existing Application from which a copy will be made.
++
+A unique [field]#application.name# is required. If present, the [field]#application.id# value will used for the new Application. If no Id is specified, a secure random UUID will be generated and used.
```

Comment:
This isn't technically accurate. The `id` is not taken from the `application.id` field in the request. It is taken from the URL. In the Create Application API request - you'll see `{applicationId}` on the URL as a request parameter. This is how we identify the Id. 

It looks like the `sourceTenantId` in the Tenant API has the same incorrect explanation. 

Perhaps we omit this part of the explanation because it is already documented on the API and using `sourceApplicationId` does not change the existing behavior. 

If we make this change, let's correct it on the Tenant API as well.